exercises/eliuds-eggs/canonical-data.json

```diff
@@ -0,0 +1,50 @@
+{
+  "exercise": "eliuds-eggs",
+  "cases": [
+    {
+      "uuid": "559e789d-07d1-4422-9004-3b699f83bca3",
+      "description": "pop-count of 0",
+      "property": "popCount",
```

Comment:
Not sure that I have a better suggestion, but does `popCount` fit the domain? I wonder if `eggCount` is a more descriptive name.

Reply:
Yup, all eggs, please. That makes it easier for users to see the connection.

Reply:
Now using eggCount for function name,